content/2020-10-15-partnership-mlh-fellowship.markdown

```diff
@@ -0,0 +1,76 @@
+Title: Kiwi TCMS is partnering with MHL Fellowship program
+Headline: for 12 weeks of open source hacking
+date: 2020-10-15 13:35
+comments: true
+og_image: images/mlh/partnership_banner.png
+twitter_image: images/mlh/partnership_banner.png
+tags: community, bounty-program
+
+
+We are happy to announce that Kiwi TCMS is going to partner with the
+[MLH Fellowship](https://fellowship.mlh.io/programs/open-source)
+open source program which is a 12 week internship alternative for
+students interested in becoming software engineers.
+
+[Major League Hacking](https://mlh.io) is the official student hackathon league.
```

Comment:
Here's a more up-to-date bio:

[Major League Hacking (MLH)](https://mlh.io) is a mission-driven B-Corp focused on empowering our next generation of technologists. Every year, more than 100,000 developers, designers, and makers join the MLH community to gain hands-on experience and build their professional networks. They are headquartered in the Greater New York Area, USA but operate world-wide.